src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
```

Comment:
The import statement for 'core.basesyntax.db.FruitTransaction' seems unnecessary here since it is not being used in the interface definition. Please remove it if it's not required.